src/java.base/share/classes/javax/net/SocketFactory.java

```diff
@@ -60,7 +60,7 @@
  *
  * <P> Factory classes are specified by environment-specific configuration
  * mechanisms.  For example, the <em>getDefault</em> method could return
- * a factory that was appropriate for a particular user or applet, and a
+ * a factory that was appropriate for a particular user, and a
```

Comment:
Suggest changing this to "particular application" instead of "particular user" as it seems a bit odd to be talking about users here.